File: sukh_1684/problems/two_sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> elements = new HashMap<>();
        int [] res = new int[2];
        for(int i=0;i<nums.length;i++){
            int r = target -nums[i];
            if(elements.containsKey(r)){
                res[0]=elements.get(r);
                res[1]= i;
            }
            else{
                elements.put(nums[i],i);
            }
        }
        return res;
    }
}

File: sukh_1684/problems/two_sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        temp = {}
        for i,val in enumerate(nums):
            if((target-val) in temp):
                return [temp[target-val],i]
            temp[val] = i
                
        